db/db.go

```diff
@@ -2467,12 +2472,12 @@ func GetWithdrawals(query string, length, start uint64, orderBy, orderDir string
 	return withdrawals, nil
 }
 
-func GetTotalAmountWithdrawn() (uint64, uint64, error) {
+func GetTotalAmountWithdrawn() (sum uint64, count uint64, err error) {
```

Comment:
Whats the point of naming the return values here, these are extra allocations?

Reply:
Only readability improvements like this:
![image](https://user-images.githubusercontent.com/114066135/220857560-14c3cb33-ed6b-4dfd-8a72-9cc32c24d9dc.png)

The two extra allocations imho have too little impact to neglect any readability improvement.
If you see this otherwise, I will revert this change.

Reply:
For future reference you can add a comment as well to document the function with additional information that will show up on hover.